src/types/resolveErrors.ts

```diff
@@ -59,6 +64,24 @@ function resolveStringsInAST(
     return ctx;
 }
 
+export function evaluateRequireErrorString(
+    expr: Ast.Expression,
+    ctx: CompilerContext,
+    util: AstUtil,
+) {
+    try {
+        return evalConstantExpression(expr, ctx, util);
+    } catch (e) {
+        if (e instanceof TactConstEvalError) {
+            throwCompilationError(
+                `The second parameter of "require()" must be evaluated at compile time`,
```

Comment:
Let's add a negative test with this error message

Reply:
Done